lib/modules/styleTweaks.js

```diff
@@ -535,114 +508,6 @@ addModule('styleTweaks', {
 			observer.observe(elem, { attributes : true, attributeFilter : ['style'] });
 		});
 	},
-	applyTopmostElementProtection: function(visible) {
-		var self = modules['styleTweaks'];
-		if (typeof visible === 'undefined') {
-			visible = true;
-		}
-		if (visible && self.srstyleHideLock.locked()) {
-			visible = false;
-		}
-
-		// great, now people are still finding ways to hide this.. these extra declarations are to try and fight that.
-		// Sorry, subreddit moderators, but users can disable all subreddit stylesheets if they want - this is a convenience
-		// for them and I think taking this functionality away from them is unacceptable.
-
-		var zIndex = 'z-index: ' + (visible ? ' 2147483646' : 'auto') + ' !important;';
-
-		self.protectedElements.each(function(index, element) {
-			var elementCss = element.getAttribute('data-res-css') || '';
-			var css = RESUtils.baseStyleProtection + zIndex + elementCss;
```

Comment:
Ditto for `baseStyleProtection` (and there are some helpers used to generate it that could also be removed)